src/main/java/com/deavensoft/paketomat/paketomats/PaketomatController.java

```diff
@@ -46,14 +46,15 @@ public int createAllPaketomatsInCountry(){
             if(c.getPopulation() > 10000){
                  numberOfPaketomats = c.getPopulation()/100000 + 1;
                 for(int i = 0; i < numberOfPaketomats; i++){
-                    Paketomat p = new Paketomat(1L,c,new ArrayList<>());
+                    Paketomat p = new Paketomat(c, Paketomat.serialNumber++);
```

Comment:
Instance methods should not write to "static" fields

Reply:
This was changed with making serialNumber private, our constructor now takes only one argument and it's city

Reply:
Why is the Center not an entity?

Reply:
We changed that